### ISSUE TITLE
phpmyadmin 5.2.1 docker image started download php file when accessing to /

### ISSUE THREAD

**Issue by guitarrapc** [bitnami/containers#68901]
### Name and Version

bitnami/phpmyadmin:5.2.1-debian-12-r32

### What architecture are you using?

arm64

### What steps will reproduce the bug?

1. Place following compose.yaml

```yaml
services:
  mysql:
    image: mysql:8.0
    ports:
      - "3306:3306"
    environment:
      - MYSQL_ROOT_PASSWORD=secret
      - MYSQL_DATABASE=phpmyadmin
      - MYSQL_USER=phpmyadmin
      - MYSQL_PASSWORD=secret
  phpmyadmin:
    image: bitnami/phpmyadmin:5.2.1-debian-12-r32
    depends_on:
      - mysql
    ports:
      - "8080:8080"
    environment:
      - PHPMYADMIN_ALLOW_ARBITRARY_SERVER=true
```

2. Run `docker compose up`
3. Open guest browse (private browse) and put `localhost:8080` to url address bar
4. You will find `download` file is downloaded instead of showing phpmyadmin login page.

### What is the expected behavior?

phpMyAdmin page should be shown when accessing to root url like `http://localhost:8080`.

### What do you see instead?

Accessing to `http://localhost:8080` download `download` file instead of showing top page. If you put `http://localhost:8080/index.php` then successfully show top page.

#### Who will affect?

Currently bitnami/phpmyadmin:5.2.1 is point to bitnami/phpmyadmin:5.2.1-debian-12-r32, so that users who cleaed host cached image `bitnami/phpmyadmin:5.2.1` encount this issue.

I've confirm both arm64 and amd64 image have same issue.

#### Workaround

Use `bitnami/phpmyadmin:5.2.1-debian-12-r31` instead of `bitnami/phpmyadmin:5.2.1` or `bitnami/phpmyadmin:5.2.1-debian-12-r32`.

#### Content-Type

When accessing with `bitnami/phpmyadmin:5.2.1-debian-12-r31` image,. top pages is `Content-Type:
text/html; charset=utf-8`, however `bitnami/phpmyadmin:5.2.1-debian-12-r32` image top pages is `Content-Type: application/x-httpd-php`.

bitnami/phpmyadmin:5.2.1-debian-12-r31

![image](https://github.com/bitnami/containers/assets/3856350/8aead996-dcc7-4a01-91ee-d9f92e44b6e9)

bitnami/phpmyadmin:5.2.1-debian-12-r32

![image](https://github.com/bitnami/containers/assets/3856350/0489994f-0581-490a-978a-463ed95d33d2)

**Comment by guitarrapc** [comment]
I suspect commit https://github.com/bitnami/containers/commit/56502da357af3f2fce3b1588c0f49d7e0ebea381 trigger issue, but can't understand why Content-Type was affected...?

**Comment by guitarrapc** [comment]
There are r33 released on 2 days ago https://github.com/bitnami/containers/commit/593cbba1f04c1c71a4de0a262e28d8e991afc0ea, and confirm issue fixed.

![image](https://github.com/bitnami/containers/assets/3856350/201794fc-ddad-45c0-8023-77482d301041)

Is this regarding to https://httpd.apache.org/security/vulnerabilities_24.html#2.4.61 which is happen from apache 2.4.60 and fixed on 2.4.61? Looking into r31, r32 and r33, apache version differs.

```sh
# r31
$ apachectl -v
Server version: Apache/2.4.59 (Unix)
Server built:   May 13 2024 14:41:26

# r32
$ apachectl -v
Server version: Apache/2.4.60 (Unix)
Server built:   Jul  1 2024 13:25:50

# r33
$ apachectl -v
Server version: Apache/2.4.61 (Unix)
Server built:   Jul  3 2024 16:36:35
```

To maintainer,
I propose unlist bitnami/phpmyadmin:5.2.1-debian-12-r32 container image, and Helm chart to prevent future impact.
